### PR TITLE
Update to leptos 0.5.1

### PR DESCRIPTION
**Pull request by lhjok**
Update to leptos 0.5.1 and resolve compatibility issues

### CODE REVIEW

File: leptonic/src/chip.rs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 use std::fmt::{Display, Formatter};
 
 use leptos::{
-    leptos_dom::{Callable, Callback},
+    Callable, Callback,
```
Comment (paulvt) [inline leptonic/src/chip.rs:4]: `Callable` and `Callback` are included in `leptos::*` by default now, so these can be removed (in this file and all the others)!

File: leptonic/src/lib.rs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 use std::fmt::Display;
 
 use leptos::{
-    leptos_dom::{Callable, Callback, StoredCallback},
+    Callable, Callback, StoredValue,
```
Comment (paulvt) [inline leptonic/src/lib.rs:4]: The same holds for `StoredValue` 🙂

**Comment by paulvt** [comment]
Could you include the same fix for the book (in `examples/book`)?

```diff
diff --git i/examples/book/Cargo.toml w/examples/book/Cargo.toml
index ed96b2d..7fcc210 100644
--- i/examples/book/Cargo.toml
+++ w/examples/book/Cargo.toml
@@ -15,7 +15,7 @@ codegen-units = 1
 console_error_panic_hook = "0.1.7"
 indoc = "2.0.1"
 leptonic = { path = "../../leptonic", features = ["csr"] }
-leptos = { version = "0.5.0-rc1", features = ["csr"] }
+leptos = { version = "0.5.1", features = ["csr"] }
 leptos-use = { git = "https://github.com/Synphonyte/leptos-use" }
 leptos_icons = { version = "0.0.16-beta", features = [
     "BsGithub",
diff --git i/examples/book/src/app.rs w/examples/book/src/app.rs
index 6e771f2..ae296f9 100644
--- i/examples/book/src/app.rs
+++ w/examples/book/src/app.rs
@@ -1,6 +1,6 @@
 use std::fmt::Display;
 
-use leptos::{leptos_dom::Callback, *};
+use leptos::*;
 use leptos_icons::BsIcon;
 use leptos_meta::{provide_meta_context, Title};
 use leptos_router::*;
```

**Comment by atcol** [comment]
Hello, is there any progress on this please? Can I help at all @lhjok @paulvt ?

**Comment by tqwewe** [comment]
Sadly it looks like this isn't working with nightly leptos

**Comment by lpotthast** [comment]
@lhjok Thanks a lot for taking the time to fix this in my absence. Have returned from a vacation now. I'm going to merge this as is and fix it up a bit.  Expect a release for 0.5 in the near future.

**Pull request merged**